Content.Server/Atmos/EntitySystems/FlammableSystem.cs

```diff
@@ -317,9 +334,19 @@ public override void Update(float frameTime)
 
                 if(transform.GridUid != null)
                 {
+                    var tile = _transformSystem.GetGridOrMapTilePosition(uid, transform);
+                    var temperature = 700f;
+                    var volume = 50f;
                     _atmosphereSystem.HotspotExpose(transform.GridUid.Value,
-                        _transformSystem.GetGridOrMapTilePosition(uid, transform),
-                        700f, 50f, true);
+                        tile,
+                        temperature, volume, true);
+
+                    // Spread the fire to other flammables on the same tile
+                    var fireEvent = new TileFireEvent(temperature, volume);
+                    foreach (var entity in _lookup.GetEntitiesIntersecting(transform.GridUid.Value, tile))
+                    {
+                        RaiseLocalEvent(entity, ref fireEvent, false);
+                    }
```

Comment:
This is a *terrible* idea. Don't raise `TileFireEvent`, only atmos internals should raise it, when the entire tile is on fire. Use physics fixtures and collisions instead, as flammable already did.

Reply:
I've spent the last two nights working with fixtures, and I can't get the papers to want to trigger a collision unless they are moving (example: when they are thrown). I want them to actively seek out a collision while they are stationary and on fire. I'm going to post a WIP here in a moment. Could you please review it and point me in the right direction?

Reply:
I give up. I'd rather push out basic paper burning now and try to figure out proper chaining later.